Blackout/components/LocationSummary.js

```diff
@@ -79,7 +80,7 @@ class LocationSummary extends Component {
             provider={PROVIDER_GOOGLE}
             showsUserLocation
             initialRegion={this.calculateViewableRegion()}
-            scrollEnabled={false}
+            scrollEnabled={true}
```

Comment:
@ajuarez99 do we want the user to be able to scroll around the map for location history or just be able to see where they went?

Reply:
yes

Reply:
should probably let them scroll around